README.md

```diff
@@ -5,3 +5,7 @@ These are our default assumed style choices unless there are existing project- o
 
   * [Python](python)
   * [R](r)
+
+---
+
+<p xmlns:cc="http://creativecommons.org/ns#" xmlns:dct="http://purl.org/dc/terms/"><span property="dct:title">The UCL-ARC coding standards guidelines</span> by <span property="cc:attributionName">the Centre for Advanced Research Computing, UCL</span> are licensed under <a href="http://creativecommons.org/licenses/by/4.0/?ref=chooser-v1" target="_blank" rel="license noopener noreferrer" style="display:inline-block;">CC BY 4.0<img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;" src="https://mirrors.creativecommons.org/presskit/icons/cc.svg?ref=chooser-v1"><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;" src="https://mirrors.creativecommons.org/presskit/icons/by.svg?ref=chooser-v1"></a></p> 
```

Comment:
It looks like GitHub at least strips a lot of the attributes out from this in rendered Markdown - not sure if Jekyll does similarly or not. Would possibly suggest to simplifying to something like
```suggestion
The UCL-ARC coding standards guidelines by the Centre for Advanced Research Computing, UCL are licensed under [CC BY 4.0](http://creativecommons.org/licenses/by/4.0/?ref=chooser-v1)<img style="height:22px;" src="https://mirrors.creativecommons.org/presskit/icons/cc.svg?ref=chooser-v1"><img style="height:22px;" src="https://mirrors.creativecommons.org/presskit/icons/by.svg?ref=chooser-v1">
```
to keep it also readable in Markdown source? But not important